android/sdk/build.gradle

```diff
@@ -137,10 +129,8 @@ gradle.projectsEvaluated {
         currentBundleTask.dependsOn("merge${buildNameCapitalized}Resources")
         currentBundleTask.dependsOn("merge${buildNameCapitalized}Assets")
 
-        runBefore("processArmeabi-v7a${buildNameCapitalized}Resources", currentBundleTask)
-        runBefore("processX86${buildNameCapitalized}Resources", currentBundleTask)
-        runBefore("processUniversal${buildNameCapitalized}Resources", currentBundleTask)
-        runBefore("process${buildNameCapitalized}Resources", currentBundleTask)
+        runBefore("package${buildNameCapitalized}Resources", currentBundleTask)
```

Comment:
The tricky part is currentBundleTask because it produces both assets and resources. Again, it can be rewritten with `variant` as above without figuring out the new name for `process${buildNameCapitalized}Resources` because `output.processResources` knows it.